heroku/index.js

```diff
@@ -9,11 +9,16 @@
 var bodyParser = require('body-parser');
 var express = require('express');
 var app = express();
+var xhub = require('express-x-hub');
+
+// use process.env.APP_SECRET from your facebook app settings
+var XHUB_SECRET = '<use app secret from your facebook app settings>';
```

Comment:
Let's document this in [graph-api-webhooks-samples/heroku/README.md](https://github.com/fbsamples/graph-api-webhooks-samples/blob/master/heroku/README.md) and just use `process.env.APP_SECRET` on line 21 to keep the configuration separate.